test/new-e2e/tests/process/linux_test.go

```diff
@@ -24,7 +24,7 @@ type linuxTestSuite struct {
 }
 
 func TestLinuxTestSuite(t *testing.T) {
-	e2e.Run(t, &linuxTestSuite{},
+	e2e.Run(t, &linuxTestSuite{}, e2e.WithSkipDeleteOnFailure(), e2e.WithDevMode(),
```

Comment:
[fix] Let's add the dev mode as an ENV var option. See: https://github.com/DataDog/datadog-agent/blob/ec97d0427f43036009c4a34c325bf9e420d2ac1d/test/new-e2e/tests/language-detection/language_detection_test.go#L44-L47

Reply:
Done